dev.heppen.webapps.json

```
@@ -0,0 +1,51 @@
+{
+  "app-id": "dev.heppen.webapps",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "24.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.rust-stable"
+  ],
+  "base": "com.system76.Cosmic.BaseApp",
+  "base-version": "stable",
+  "command": "quick-webapps",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--share=network",
+    "--filesystem=host-os:ro",
+    "--filesystem=xdg-data/applications",
+    "--filesystem=xdg-data/flatpak",
```

```suggestion
    "--filesystem=xdg-data/flatpak",
```

this should be `xdg-data/flatpak/exports/share/applications:ro` shouldn't it? You don't install desktop files there, only read.

---

Ah those are symlinks I forgot

---

Adding the target should fix it `xdg-data/flatpak/app:ro`, `dg-data/flatpak/exports/share/applications:ro`

---

Yeah i thought that's the reason and tried different ways to achieve this but with this permission it's just easier :P

---

you can keep `xdg-data/flatpak` but this should be :ro anyways

---

oh yeah, it works with `ro`, I don't know, I'm pretty sure I tested it before and it did not, haha. Ok, I will change this :D